lib/payment_processor/go_cardless/transaction.rb

```diff
@@ -71,7 +70,9 @@ def action_params
           currency:         currency,
           transaction_id:   transaction_id,
           is_subscription:  false,
-          payment_provider: 'go_cardless'
+          payment_provider: 'go_cardless',
```

Comment:
I think it might be useful to persist this data locally, as well. At least for the purpose of removing payment methods for the member management sprint. E.g. "Remove this mandate you have stored for this bank account at this bank."